composer.json

```diff
@@ -0,0 +1,7 @@
+{
+    "name": "shopgate/cart-integration-magento2-shopgate-bolt",
```

Comment:
Just saw that we have 2 times shopgate in the name (as well as in the repository name). You can just remove the second one to have shopgate/cart-integration-magento2-bolt instead of shopgate/cart-integration-magento2-shopgate-bolt

Reply:
repository name and composer name have been updated.